plugins/edc-build/src/main/java/org/eclipse/edc/plugins/edcbuild/conventions/TestConvention.java

```diff
@@ -25,24 +26,45 @@
  */
 class TestConvention implements EdcConvention {
     private static void determineJunitPlatform(Test testTask) {
+        // parse task exclusion
+        var excludedTagsProperty = System.getProperty("excludeTags");
+        var excludedTags = getTags(excludedTagsProperty);
+
+
         // Target all type of test e.g. -DrunAllTests="true"
         var runAllTests = Boolean.parseBoolean(System.getProperty("runAllTests", "false"));
         if (runAllTests) {
-            testTask.useJUnitPlatform();
+            // honor excluded tags -> blacklisting
+            if (excludedTags.length > 0) {
+                testTask.useJUnitPlatform(platform -> platform.excludeTags(excludedTags));
+            } else {
+                testTask.useJUnitPlatform();
+            }
         } else {
             var includeTagsProperty = System.getProperty("includeTags");
-            var tags = ofNullable(includeTagsProperty)
-                    .map(prop -> prop.split(","))
-                    .orElse(new String[0]);
+            var includedTags = getTags(includeTagsProperty);
 
-            if (tags.length > 0) {
-                testTask.useJUnitPlatform(platform -> platform.includeTags(tags));
+            // white-list included tags...
+            if (includedTags.length > 0) {
+                testTask.useJUnitPlatform(platform -> platform.includeTags(includedTags));
+                //... and possibly black-list excluded tags
+                if (excludedTags.length > 0) {
```

Comment:
what is the case where someone wants to include and exclude tags in the same execution?
I see two cases:
- default (no tags included) -> possibility to include
- runAllTests (all tags included) -> possibility to exclude

Reply:
this could happen on "derived" tags, e.g. someone includes `ComponentTest` but wants to exclude `AVerySpecialComponentTest`